PORTAL/jobs/__main__.py

```diff
@@ -1015,7 +1019,8 @@ def main(cmd, cmd_kwargs, cfgfile=None, user=None, devmode=False):
     logger.debug('# loading config from %s', cfgfile)
     cfg = JobsConfig.load(cfgfile)
 
-    jobs = Jobs(cfg, devmode=devmode)
+    jobs = Jobs(cmd_kwargs['worker'], cfg, devmode=devmode)
+    cmd_kwargs.pop('worker')
```

Comment:
This change goes away if we keep the worker specific to the commands where it's relevant.